src/v0/destinations/sendinblue/data/SendinblueTrackLinkConfig.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "destKey": "email",
+    "sourceKeys": ["context.traits.email", "properties.email"]
+  },
+  {
+    "destKey": "phone",
+    "sourceKeys": ["context.traits.phone", "properties.phone"]
+  },
+  {
+    "destKey": "link",
+    "sourceKeys": "properties.link",
```

Comment:
what does this link signify. I remember we had some hiccups while implementing this in device mode. How this one is different? Can't we map context.page.url here?

Reply:
- Track link is not implemented in device mode.
- As, per their doc -> `Track link is used to track click events on various links of your website.` . 
- I think we can't map `context.page.url` to `link`.  As this is page url.